great_expectations/data_context/types/base.py

```diff
@@ -976,7 +982,7 @@ class Meta:
     module_name = fields.String(
         required=False,
         allow_none=True,
-        load_default="great_expectations.execution_engine",
+        missing="great_expectations.execution_engine",
```

Comment:
Looks like we forgot to revert this use of `missing` as part of https://github.com/great-expectations/great_expectations/pull/6271